### PR TITLE
Changelog for 19.16.2.2

### PR DESCRIPTION
**Pull request by akuzm**
Category (leave one):
- Documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,248 @@
...
+[#7294](https://github.com/ClickHouse/ClickHouse/pull/7294) ([hcz](https://github.com/hczhcz))
+* Add new functions that return the Array of all matched indices in multiMatch family of functions.
+  [#7299](https://github.com/ClickHouse/ClickHouse/pull/7299) ([Danila
+Kutenin](https://github.com/danlark1))
+* Add a new database engine `Lazy` that is optimized for storing a large number of small -Log
+  tables.  [#7171](https://github.com/ClickHouse/ClickHouse/pull/7171) ([Nikita
+Vasilev](https://github.com/nikvas0))
+* Add AND, OR, XOR aggregate function for bitmap columns. [#7109](https://github.com/ClickHouse/ClickHouse/pull/7109) ([Zhichang
+Yu](https://github.com/yuzhichang))
+* Add aggregate function combinators which fill null or default value when there is nothing to
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: Better to specify their names.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,248 @@
...
+([alesapin](https://github.com/alesapin))
+* Fix wrong result for some queries given by the optimization of empty IN subqueries and empty
+  INNER/RIGHT JOIN. [#7284](https://github.com/ClickHouse/ClickHouse/pull/7284) ([Nikolai
+Kochetov](https://github.com/KochetovNicolai))
+
+### Improvement
+* Add a message in case of queue_wait_max_ms wait takes place.
+  [#7390](https://github.com/ClickHouse/ClickHouse/pull/7390) ([Azat
+Khuzhin](https://github.com/azat))
+* Minor improvements of Template format. [#7118](https://github.com/ClickHouse/ClickHouse/pull/7118)
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: That PR added a new feature.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,248 @@
...
+### Bug Fix
+* Fix wrong query result if it has `WHERE IN (SELECT ...)` section and `optimize_read_in_order` is
+  used. [#7371](https://github.com/ClickHouse/ClickHouse/pull/7371) ([Anton
+Popov](https://github.com/CurtizJ))
+* Disabled MariaDB authentication plugin, which depends on files outside of project.
+  [#7140](https://github.com/ClickHouse/ClickHouse/pull/7140) ([Yuriy
+Baranov](https://github.com/yurriy))
+* Fix exception `Cannot convert column ... because it is constant but values of constants are
+  different in source and result` which could rarely happen when functions `now()`, `today()`,
+`yestarday()`, `randConstant()` are used.
```
Comment (den-crane) [inline CHANGELOG.md]: typo 
-yestarday()
+yesterday()

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,248 @@
...
+  [#7359](https://github.com/ClickHouse/ClickHouse/pull/7359) ([Amos
+Bird](https://github.com/amosbird))
+* Return early if the subquery is empty.
+  [#7007](https://github.com/ClickHouse/ClickHouse/pull/7007) ([小路](https://github.com/nicelulu))
+* Optimize parsing of SQL expression in Values.
+  [#6781](https://github.com/ClickHouse/ClickHouse/pull/6781)
+([tavplubix](https://github.com/tavplubix))
+
+### Build/Testing/Packaging Improvement
+* Disable some contribs for cross-compilation.
```
Comment (alesapin) [inline CHANGELOG.md]: for `macOS`?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,248 @@
...
+* Fix undefined behavior in murmurhash32. [#7388](https://github.com/ClickHouse/ClickHouse/pull/7388) ([Amos
+  Bird](https://github.com/amosbird))
+* Fix undefined behavior in StoragesInfoStream. [#7384](https://github.com/ClickHouse/ClickHouse/pull/7384)
+  ([tavplubix](https://github.com/tavplubix))
+* Fixed constant expressions folding for external database engines (MySQL, ODBC, JDBC). In previous
+  versions it wasn't working for multiple constant expressions and was not working at all for Date,
+DateTime and UUID. This fixes [#7245](https://github.com/ClickHouse/ClickHouse/issues/7245)
+[#7252](https://github.com/ClickHouse/ClickHouse/pull/7252)
+([alexey-milovidov](https://github.com/alexey-milovidov))
+* Fixing AddressSanitizer error in the LIVE VIEW getHeader() method.
```
Comment (alesapin) [inline CHANGELOG.md]: bugfix?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,248 @@
...
+  [#7271](https://github.com/ClickHouse/ClickHouse/pull/7271)
+([vzakaznikov](https://github.com/vzakaznikov))
+* Wait for all scheduled jobs, which are using local objects, if `ThreadPool::schedule(...)` throws
```
Comment (alesapin) [inline CHANGELOG.md]: Code improvement?

**Pull request merged**